com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -683,6 +684,12 @@ public int colorGradingLutSize
             set { m_ColorGradingLutSize = Mathf.Clamp(value, k_MinLutSize, k_MaxLutSize); }
         }
 
+        public bool useFastSRGBLinearConversion
```

Comment:
All public API needs API docs, or consider making it internal/private if public is not intended.